### ISSUE TITLE
chromosome ploidy labels

### ISSUE THREAD

**Issue by mrouard** [zeleniy/ideogram#21]
Could you please add a cross-check function between the label of the ancestral genomes and the label displayed for each chromosomes?

In some cases, one may want to label a chromosome like AB1B2 or ABB’ to distinguish between 2 ancestral B. Therefore, it would be good
that in the example below 'AB1B2' is considered like 3  chromosomes because it is specified in the anscestors. At the moment, I think it considers there are 5, chromosomes (A, B, 1, B, 2).

      ancestors: {
        "A": "#dea673",
        "B1": "#7396be"
        "B2": "green"

      },
      ploidyDesc: [
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2',
        'AB1B2'
      ]
    };

**Comment by zeleniy** [comment]
Yes, you are right. It support now only one-letter ancestor's description. Will extend this functionality soon.
